### PR TITLE
KakaoBrain Custom Resnet9 || CIFAR10 || Kakao Brain

### PR DESCRIPTION
**Pull request by wbaek**
We updated the training and inference recodes

### CODE REVIEW

File: CIFAR10/train/kakaobrain_custom-resnet9_4v100-braincloud_pytorch.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
     "author": "clint@KakaoBrain",
     "authorEmail": "clint.b@kakaobrain.com",
     "framework": "PyTorch 1.1.0",
-    "codeURL": "https://github.com/wbaek/torchskeleton/releases/tag/v0.1.1_dawnbench_cifar10_release",
+    "codeURL": "https://github.com/wbaek/torchskeleton/releases/tag/v0.2.1_dawnbench_cifar10_release",
     "model": "Kakao Brain Custom ResNet9",
     "hardware": "Tesla V100 * 4 GPU / 488 GB / 56 CPU (Kakao Brain BrainCloud)",
-    "timestamp": "2019-08-05",
+    "timestamp": "2019-08-22",
```
Comment (codyaustun) [inline CIFAR10/train/kakaobrain_custom-resnet9_4v100-braincloud_pytorch.json]: I can't verify this date. Please change to the date this PR was submitted.

File: CIFAR10/inference/kakaobrain_custom-resnet9_1v100-braincloud_pytorch_inference.json [hunk 2]
```diff
@@ -3,12 +3,12 @@
...
+    "codeURL": "https://github.com/wbaek/torchskeleton/releases/tag/v0.2.1_dawnbench_cifar10_release",
     "commitHash": "96c0cbf0bb9ce0734512048a124caf82bf24c392",
     "model": "Kakao Brain Custom ResNet9 using PyTorch JIT in python",
     "hardware": "Tesla V100 * 1 GPU / 488 GB / 56 CPU (Kakao Brain BrainCloud)",
-    "latency": 0.85933,
-    "top1Accuracy": 94.20,
-    "timestamp": "2019-08-06",
+    "latency": 0.85702,
+    "top1Accuracy": 94.23,
+    "timestamp": "2019-08-22",
```
Comment (codyaustun) [inline CIFAR10/inference/kakaobrain_custom-resnet9_1v100-braincloud_pytorch_inference.json]: I can't verify this date. Please change to the date this PR was submitted.

**Review by codyaustun — CHANGES_REQUESTED** [review]
Looks good overall, but the date seems wrong.

**Comment by wbaek** [comment]
@codyaustun  we have updated timestamps.

**Comment by codyaustun** [comment]
Looks good to me!

**Pull request merged**